packages/near-fast-auth-signer/src/components/AddDevice/AddDevice.tsx

```diff
@@ -84,6 +84,7 @@ function AddDevicePage() {
   const defaultValues = {
     email: searchParams.get('email') ?? '',
```

Comment:
Not major, but I think we need to display a message. 

If the user remove all keys from the account on Remove Keys Screen, then try to login again with the e-mail he deleted the keys, the Login will fail but there is no error message. 

Then, if the user, do the flow with the email it will work fine (He goes to verify e-mail and it works)

Reply:
The fix for this is to not allow the user to delete the current signer instance's FAK. Can be addressed in a follow up